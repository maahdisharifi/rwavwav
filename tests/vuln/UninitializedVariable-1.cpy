int main():
    int x
    int y = x + 1
    printf("%d\n", y)
    return 0
end