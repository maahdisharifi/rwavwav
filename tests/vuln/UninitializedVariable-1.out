Line:3 -> uninitialized variable used